BuildTools/swiftformat.sh

```diff
@@ -9,8 +9,11 @@ if [ -z "$CI" ]; then
     SWIFT_FORMAT_VERSION=$(grep -F $SWIFT_FORMAT $MINT_FILE_PATH)
 
     if ! xcrun --sdk macosx mint which -s $SWIFT_FORMAT_VERSION > /dev/null; then
-      xcrun --sdk macosx mint bootstrap -m $MINT_FILE_PATH
+        xcrun --sdk macosx mint bootstrap -m $MINT_FILE_PATH
     fi
 
-    xcrun --sdk macosx mint run -m $MINT_FILE_PATH $SWIFT_FORMAT --config "$BASEDIR/.swiftformat" .
+    git diff --diff-filter=d --staged --name-only | grep -e '\(.*\).swift$' | while read file; do
```

Comment:
Nit: I think Swiftlint also formats other files, for example localizable.string, do you think it is important to keep it this way?

Reply:
I'll check how that behaves now and how it behaves with the changes before merging - thanks for pointing that out!

Reply:
I just checked the localization files, and the lint doesn't apply to them.